Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -556,94 +589,3 @@ extension NewsTableViewController: UIViewControllerTransitioningDelegate {
         return DimPresentAnimationController()
     }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/1834ef044384d2d76bab6a6262228a3ff2bb02ec/Headlines/src/Controllers/NewsTableViewController.swift#L591"></span>File should contain 400 lines or less: currently contains 591<br />
<code>file_length</code> <code>NewsTableViewController.swift:591</code></td>
    </tr>
  </tbody>
</table>